app/decorators/pseud_decorator.rb

```diff
@@ -90,16 +90,16 @@ def fandom_link(fandom_id)
   end
 
   def authored_items_links(options = {})
+    general_links = [works_link, bookmarks_link].compact.join(", ")
     if options[:fandom_id].present?
       # This can potentially be an array
-      links = [options[:fandom_id]].flatten.map do |fandom_id|
+      fandom_links = [options[:fandom_id]].flatten.map do |fandom_id|
         fandom_link(fandom_id)
       end
-      links << bookmarks_link
+      general_links + " | " + fandom_links.compact.join(", ")
```

Comment:
Do we need to handle cases where either `general_links` or `fandom_links` is empty? This doesn't affect the old code since we only did a join in the end.

Reply:
I don't think so. I didn't run into any errors when search for pseuds who had no works and/or no bookmarks, and if `fandom_links` is blank, the pseud won't be included in the results for a fandom search.